nussknacker-dist/src/universal/conf/application.conf

```diff
@@ -40,3 +40,12 @@ countsSettings {
   influxUrl: ${?COUNTS_URL}
   database: "esp"
 }
+
+kafka {
+  kafkaAddress: ${?KAFKA_ADDRESS}
+  kafkaProperties {
+    "schema.registry.url": ${?SCHEMA_REGISTRY_URL}
+  }
+  consumerGroupNamingStrategy: processId-nodeId
```

Comment:
it is default strategy - you can remove it. this application.conf should be as smallest as possible

Reply:
removed from here